feature/dialog/src/main/java/com/terning/feature/dialog/cancel/ScrapCancelViewModel.kt

```diff
@@ -2,6 +2,7 @@ package com.terning.feature.dialog.cancel
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.terning.core.designsystem.R.string.server_failure
```

Comment:
`as`라는 예약어를 사용하여 라이브러리의 이름을 지정하고 사용할 수 있어요!

```suggestion
import com.terning.core.designsystem.R as DesignSystemR
```

위와 같이 라이브러리에 이름을 부여하면 코드에선 `DesignSystemR.string.server_failure`로 사용이 가능합니다!

Reply:
오호 이에 대해선 몰랐네요!! 이 방법이 헷갈리지 않고 쓸 수 있는 것 같아요! 적용해보겠습니당